src/VIP/Admin/CreateOffer.js

```diff
@@ -275,14 +292,23 @@ class CreateOffer extends React.Component {
                     type="date"
                     placeholder="start date"
                     onChange={this.onStartDateChange.bind(this)}
+                    isInvalid={this.state.sDateNull}
                   />
+                  <Form.Control.Feedback type="invalid">Start date is required</Form.Control.Feedback>
                 </Col>
-                <Col sm={3}>
-                  <Form.Control controlId="endDate" type="date" placeholder="end date" onChange={this.onEndDateChange.bind(this)} isInvalid={this.state.dateStatus}/>
-                  <Form.Control.Feedback type='invalid'>
-                    start-date should be before end-date
-                  </Form.Control.Feedback>
-                </Col>
+                  <Col sm={3}>                  
+                    <Form.Control 
+                      controlId="endDate" 
+                      type="date" 
+                      placeholder="end date" 
+                      onChange={this.onEndDateChange.bind(this)} 
+                      isInvalid={this.state.eDateNull || this.state.dateStatus}
+                    />
+                    <Form.Control.Feedback type='invalid'> 
+                      End date is required. Start-date should be before end-date
```

Comment:
I suggest this sentence might be "Start-date should be before end-date" enough.

Reply:
Not mandantory suggestion :)

Reply:
Oh I will fix like that now